web/views.py

```
@@ -6949,3 +6951,168 @@ def delete_post(request, post_id):
+        - Surprising or not commonly known
+        - Educational and thought-provoking
+        - Appropriate for all ages
+        - Scientifically accurate (if applicable)
+        - 22 words or less
+        - Include one relevant emoji
+        Respond with ONLY the fact, no prefixes like "Did you know:" or similar."""
+        response = model.generate_content(prompt, generation_config={"temperature": 0.9})
+        return response.text.strip()
+    except Exception:
+        logger = logging.getLogger(__name__)
+        logger.exception(f"Error generating fact with Gemini for subject '{subject_name}'")
```

Ruff (0.8.2)

Logging statement uses f-string

(G004)